frontend/src/components/camp/picasso/Picasso.vue

```diff
@@ -287,13 +339,19 @@ export default {
       this.entryWidth = Math.max((this.$refs.calendar.$el.offsetWidth - widthIntervals) / this.$refs.calendar.days.length, 80)
     },
     getActivityName (scheduleEntry, _) {
+      if( scheduleEntry.tmpEvent ) return this.$tc('entity.activity.new')
```

Comment:
lässt der linter das durch?

Reply:
Ne 😄 
Mein Autolinter in der IDE war offensichtlich deaktiviert.